build.gradle

```diff
@@ -240,7 +240,7 @@ project(':core') {
 
   dependencies {
     compile project(':clients')
-    compile project(':log4j-appender')
+    compile "$slf4jlog4j"
```

Comment:
Added slf4j, log4j jar dependency to core project.  This is to avoid old version of transitive dependencies from Zookeeper, ZKClient dependencies.  ZK and ZKClient depends on older version of slf4j and log4j.